modulefiles/modulefile.hafs.hera

```diff
@@ -0,0 +1,71 @@
+#%Module#####################################################
+## Module file for hafs
+#############################################################
+proc ModulesHelp { } {
+    puts stderr "Set environment veriables for compiling or running HAFS"
+}
+module-whatis " HAFS model prerequisites"
+
+module use /contrib/sutils/modulefiles
+module load sutils
+module load hpss
+
+module load cmake/3.20.1
+setenv CMAKE_C_COMPILER mpiicc
+setenv CMAKE_CXX_COMPILER mpiicpc
+setenv CMAKE_Fortran_COMPILER mpiifort
+setenv CMAKE_Platform hera.intel
+
+module use /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/modulefiles/stack
+module load hpc/1.1.0
+module load hpc-intel/18.0.5.274
+module load hpc-impi/2018.0.4
+
+module load jasper/2.0.22
+module load zlib/1.2.11
+module load png/1.6.35
+
+module load hdf5/1.10.6
+module load netcdf/4.7.4
+module load pio/2.5.2
+module load esmf/8_1_1
+module load fms/2020.04.03
+
+module load bacio/2.4.1
+module load crtm/2.3.0
+module load g2tmpl/1.10.0
+module load ip/3.3.3
+module load nemsio/2.5.2
+module load sp/2.3.3
+module load w3emc/2.7.3
+module load w3nco/2.4.1
+module load upp/10.0.8
+
+module load gftl-shared/v1.3.0
+module load yafyaml/v0.5.1
+module load mapl/v2.7.3
+
+module load g2/3.4.3
+
+module load bufr/11.4.0
+module load gfsio/1.4.1
+module load landsfcutil/2.4.1
+module load nemsiogfs/2.5.3
+module load sfcio/1.4.1
+module load sigio/2.3.2
+module load szip/2.1.1
+module load wrf_io/1.1.1
+
+module load prod_util/1.2.2
+module load grib_util/1.2.2
+
+module load wgrib2/2.0.8
+setenv WGRIB2 /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/intel-18.0.5.274/impi-2018.0.4/wgrib2/2.0.8/bin/wgrib2
+
+module use /scratch2/NCEPDEV/nwprod/NCEPLIBS/modulefiles
+module load nco/4.9.1
+
+module use -a /scratch1/NCEPDEV/nems/emc.nemspara/soft/modulefiles
+module load rocoto/1.3.3
+
+module load intelpython/3.6.8
```

Comment:
@BijuThomas-NOAA, it would also be better to remove all the other modules not needed to build/run GFDL tracker. Thanks!